channels/alpha

```diff
@@ -17,6 +17,9 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
+  - range: ">=1.7.0"
```

Comment:
I'm happy to leave it out, but there's the issue of having addons not change unless the channel version changes.  It's probably not a problem for this PR though, for now we can't make an upgrade work anyway.  I can remove.